commons/utils/src/main/java/in/projecteka/jataayu/util/ui/DateTimeUtils.kt

```diff
@@ -23,9 +24,15 @@ class DateTimeUtils {
         }
 
         fun getDate(utcDate: String): Date? {
-            val inputFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS", Locale.getDefault())
+            var inputFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS", Locale.getDefault())
             inputFormat.timeZone = TimeZone.getTimeZone("UTC")
-            return inputFormat.parse(utcDate)
+            return try {
+                inputFormat.parse(utcDate)
+            } catch (exception: ParseException) {
+                inputFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.getDefault())
+                inputFormat.timeZone = TimeZone.getTimeZone("UTC")
+                inputFormat.parse(utcDate)
+            }
```

Comment:
what if this one error exception? :(

https://stackoverflow.com/questions/4024544/how-to-parse-dates-in-multiple-formats-using-simpledateformat

Can you do something like this please?